src/gui_common/tooltip/ToolTipManager.tscn

```diff
@@ -4040,6 +4040,12 @@ margin_right = 345.0
 margin_bottom = 203.0
 mouse_filter = 2
 
+[node name="to be implemented" type="Label" parent="GroupHolder/organelleSelection/thermoplast/MarginContainer/VBoxContainer"]
```

Comment:
Prefer using `PascalCase` for node names, since it's widely used for nodes in various scenes.

Reply:
I think it would be possible to add a style check to make sure each node name attribute contains at least one capital letter and no spaces. I'll open an issue: https://github.com/Revolutionary-Games/Thrive/issues/2150

Reply:
So what should the node be called?

Reply:
`ToBeImplemented` is also what I had in mind.